src/Akeneo/Connectivity/Connection/back/tests/Integration/Persistence/Dbal/Query/DbalSelectErrorCountPerConnectionQueryIntegration.php

```diff
@@ -72,7 +72,7 @@ public function test_it_gets_error_count_per_connection()
             new ErrorCount('no_error', 0),
         ]);
 
-        Assert::assertEquals($expectedResult, $result);
+        $this->assertEqualsCanonicalizing($expectedResult, $result);
```

Comment:
```suggestion
        Assert::assertEqualsCanonicalizing($expectedResult, $result);
```
(it's a static method :wink: )